src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -52,10 +52,10 @@
 #include "mongoBackend/connectionOperations.h"
 #include "mongoBackend/TriggeredSubscription.h"
 #include "cache/subCache.h"
-
+#include "rest/StringFilter.h"
 #include "ngsi/Scope.h"
 #include "rest/uriParamNames.h"
-
+#include "rest/StringFilter.h"
```

Comment:
It is not C/C++, but the C preprocessor.
And cpp is pretty stupid ...
However there is the good old #ifndef/#define/#endif   or, even better, #pragma once  to avoid double inclusion.

To include the file twice is a mistake of course and removed in the next commit.